docs/policy-packages.md

```diff
@@ -41,34 +41,35 @@ the Rucio codebase. These can be found in
 `lib/rucio/common/schema/generic.py` respectively.
 
 In the has_permission function you may wish to default to
-the generic permission checks if your experiment does not need specific
+the generic permission checks04aadey if your experiment does not need specific
```

Comment:
?

Reply:
My bad this is from my side 😓 .. Sorry will fix this 😄